app/styles/teachers/teacher-student-view.sass

```diff
@@ -3,6 +3,30 @@
 @import "app/styles/style-flat-variables"
 
 #teacher-student-view
+  #numerical-courses-status
```

Comment:
We have almost the same styles in #edit-student-modal.sass, if you can think of a convenient way to reafctor the common styles out. Not necessary.